### PR TITLE
Gradle 4.4 and some Readme badges

### PR DESCRIPTION
**Pull request by amdelamar**
Hi - Just wanted to help out by adding a release version badge, and license badge to the README.

Also, updated Gradle wrapper's version to `4.4`. 
I did `./gradlew clean build` and didn't have any issues. But please try on your end as well.

Thanks

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2,14 +2,18 @@
 
 An _opinionated_ Gradle plugin for Vert.x projects.
 
+image:https://img.shields.io/badge/release-0.0.6-green.svg[release, link=https://plugins.gradle.org/plugin/io.vertx.vertx-plugin]
```
Comment (jponge) [inline README.adoc]: Can we have a version-agnostic link to follow versions without having to editing the `README` file?

Comment (amdelamar) [reply README.adoc]: Yeah that was my initial thought too. But it doesn't seem to work on your GitHub Releases for some reason. Open this in your browser: https://img.shields.io/github/release/jponge/vertx-gradle-plugin.svg For me it shows "release: none"... so this was my workaround for now.

Comment (spodin) [reply README.adoc]: Seems, that latest version resolves using GitHub Releases, not tags. As I know, there isn't any way get badge from Gradle Plugins directly, so for tags as versions can recommend to use: `image:https://badge.fury.io/gh/jponge%2Fvertx-gradle-plugin.svg["GitHub version", link="https://plugins.gradle.org/plugin/io.vertx.vertx-plugin"]`

Comment (amdelamar) [reply README.adoc]: Ok, added "tag" badge to grab the latest tag version instead.

**Comment by amdelamar** [comment]
Up to @jponge if the "version/tag" badge should link to Gradle Plugin page, GitHub releases page, or no link at all. I'd think users should see it link to `plugins.gradle.org` for validity that this is the very same plugin code. And license is of course optional fluff.

**Comment by jponge** [comment]
Git tags are a good indicator of what is the latest version so that badge
is useful.

The LICENSE one is more a decorative one :-)

**Comment by jponge** [comment]
Thanks!

**Pull request merged**